roles/pulibrary.timezone-eastern/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Sets eastern timezone for cron jobs
+  command: timedatectl set-timezone America/New_York
```

Comment:
Does this set the timezone for cron or the whole system?

Reply:
Yes it does.